zebra-chain/src/transaction/serialize.rs

```diff
@@ -21,7 +21,7 @@ use crate::{
 };
 
 use super::*;
-use sapling::{Output, SharedAnchor, Spend};
+use crate::sapling::SharedAnchor;
```

Comment:
Nitpick/Optional: Could we revert the other changes in this file and update this to:

```suggestion
use crate::sapling::{Output, SharedAnchor, Spend};
```

Reply:
Thanks, fixed this at https://github.com/ZcashFoundation/zebra/pull/8522/commits/1da71d8888782d25f82118f6810421075f31c246